doc/source/contributing.rst

```diff
@@ -0,0 +1,330 @@
+
+This will create the new environment, and not touch any of your existing environments,
+nor any existing python installation. 
+
+To work in this environment, Windows users should ``activate`` it as follows::
+
+      activate geopandas_dev
+
+Mac OSX and Linux users should use::
+
+      source activate geopandas_dev
+
+You will then see a confirmation message to indicate you are in the new development environment.
+
+To view your environments::
+
+      conda info -e
+
+To return to you home root environment::
+
+      deactivate
+
+See the full conda docs `here <http://conda.pydata.org/docs>`__.
+
+At this point you can easily do a *development* install, as detailed in the next section.
+
+.. _contributing.windows:
+
+Creating a Windows development environment
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To build on Windows, you need to have compilers installed to build the extensions. You will need to install the appropriate Visual Studio compilers, VS 2008 for Python 2.7, VS 2010 for 3.4, and VS 2015 for Python 3.5.
+
+For Python 2.7, you can install the ``mingw`` compiler which will work equivalently to VS 2008::
+
+      conda install -n geopandas_dev libpython
+
+or use the `Microsoft Visual Studio VC++ compiler for Python <https://www.microsoft.com/en-us/download/details.aspx?id=44266>`__. Note that you have to check the ``x64`` box to install the ``x64`` extension building capability as this is not installed by default.
+
+For Python 3.4, you can download and install the `Windows 7.1 SDK <https://www.microsoft.com/en-us/download/details.aspx?id=8279>`__. Read the references below as there may be various gotchas during the installation.
+
+For Python 3.5, you can download and install the `Visual Studio 2015 Community Edition <https://www.visualstudio.com/en-us/downloads/visual-studio-2015-downloads-vs.aspx>`__.
+
+Here are some references:
+
+- https://github.com/conda/conda-recipes/wiki/Building-from-Source-on-Windows-32-bit-and-64-bit
+- https://cowboyprogrammer.org/building-python-wheels-for-windows/
+- https://blog.ionelmc.ro/2014/12/21/compiling-python-extensions-on-windows/
+- https://support.enthought.com/hc/en-us/articles/204469260-Building-Python-extensions-with-Canopy
+
+
+
+
+3) Installing Dependencies
+---------------------------
+
+To run *geopandas* in an development environment, you must first install *geopandas*'s dependencies. We suggest doing so using the following commands (executed after your development environment has been activated):
+
+    conda install fiona -c conda-forge
+    conda install shapely -c conda-forge
+    conda install pyproj -c conda-forge
+    conda install pandas
+
+
+This should install all necessary dependencies. The only exception is `rtree`_, which is an optional dependency. ``rtree`` requires the C library `libspatialindex`_. If using brew, you can install using ``brew install Spatialindex``.
```

Comment:
Can you update this? I know it was already there, but I think it is recommended to install this with conda as well